pysys/baserunner.py

```diff
@@ -100,7 +104,8 @@ def __init__(self, record, purge, cycle, mode, threads, outsubdir, descriptors,
 		@param record: Indicates if the test results should be recorded 
 		@param purge: Indicates if the output subdirectory should be purged on C{PASSED} result
 		@param cycle: The number of times to execute the set of requested testcases
-		@param mode: The user defined mode to run the testcases in
+		@param mode: Only used if supportMultipleModesPerRun=False; specifies the single mode 
+		tests will be run with. 
 		@param threads: The number of worker threads to execute the requested testcases
 		@param outsubdir: The name of the output subdirectory
 		@param descriptors: List of XML descriptor containers detailing the set of testcases to be run
```

Comment:
These are no longer specifically XML descriptors according to the changed comment above.

Reply:
good point, fixed now